### ISSUE TITLE
journal

### ISSUE THREAD

**Issue by fsolt** [fsolt/dcpo_trust_bureaucracy#4]
We cite the following PA journals (rankings by [scimago](https://www.scimagojr.com/journalrank.php?category=3321)):

Public Administration Review (1947, 2006, 2015; rank: 2)
Journal of Public Administration Research and Theory (2011; rank: 3)
Public Administration (2011; rank: 12)
Administration & Society (1998, 2005; rank: 33)
International Journal of Public Administration (2016; rank: 68)
Public Administration Quarterly (2021; rank: 111)
Administration (2012; rank: 137)

I _think_ that even if this qualifies as some of our best work (and I do like it), it is not in the heart of either of our research programs, so it doesn't qualify as an "aim high."  (I would be glad to reconsider if you tell me that I am mistaken on this.  Seriously, no problem.)  That rules out _PAR_ and _JPART_ and probably even _PA_.  I _suspect_ that _A&S_ isn't a good fit for us, just based on the name and the titles of the things we cite.  In principle, I'd be happy with any of the others.  Maybe start with _IJPA_ and work down as needed?  Anyway, I agree that your idea of looking at recent issues of these journals to see if there's anything else we might cite (and so help us decide) is a good one.

**Comment by Tyhcass** [comment]
@fsolt  No, this is not a piece aiming high. Among PA journals, the International Journal of Public Administration seems like a potential option, but its recent publications primarily focus on topics like organizational administration and corporate management, which are not closely aligned with our work.

We could consider submitting to IJPA or explore PS journals similar to Social Science Quarterly (SSQ).

**Comment by fsolt** [comment]
> we could ... explore PS journals similar to Social Science Quarterly (SSQ)

Yeah, "a place like SSQ" would be perfect for us, given that we aren't actually PA people, but I don't know where that would be besides SSQ, and I think we probably don't want to go there again so soon?  To throw out some names plucked from [Garand et al. 2009](https://doi.org/10.1017/S1049096509990205): _Political Studies_, _Polity_, _Governance_, _Journal of European Public Policy_.  Hmm, [that last](https://www.tandfonline.com/action/showAxaArticles?journalCode=rjpp20) sounds maybe extra-promising.  Want to take a look?  If they have anything we can cite, drop that in, and then send it off?  If not, or you otherwise think it is a poor fit, check out the others.

**Comment by fsolt** [comment]
![image](https://github.com/user-attachments/assets/1aa90679-729f-41bf-b1c9-40447b0654d1)

As much as I love the idea of doing a Euro version of this--we have not yet taken advantage of the chance to do regional work (and it is there)--"in Europe and elsewhere" would seem to cover it.

**Comment by Tyhcass** [comment]
Let's target JEPP, and I’ll add some citations from it.

**Comment by Tyhcass** [comment]
@fsolt I've added citations from JEPP and reframed the intro and conclusion to emphasize how TCS could contribute to public policy studies.  Any suggestions are welcome.

**Comment by fsolt** [comment]
I just submitted the working paper version to SocArXiv at https://osf.io/preprints/socarxiv/vjuah

Will you submit it to JEPP, please?

**Comment by Tyhcass** [comment]
@fsolt It has been submitted to JEPP!
